tests/particle/test_decfilenames.py

```diff
@@ -723,26 +729,7 @@
 
 # Sub-list of particle names that Particle.from_dec has trouble with at present
 # Consider this as a TODO
-list_undealt_with = [
- "D'_s1+",
- "D'_s1-",
- "K''*+",
- "K''*-",
- "K''*0",
- "K'*+",
- "K'*-",
- "K'*0",
- "K'_1+",
- "K'_1-",
- "K'_10",
- "Xi'_b-",
- "Xi'_b0",
- "anti-K''*0",
- "anti-K'*0",
- "anti-K'_10",
- "anti-Xi'_b+",
- "anti-Xi'_b0"
-]
+list_undealt_with = []
```

Comment:
This could probably be removed now, right?

Reply:
In fact the tests may well need to be "refactored". But at least this can be removed, as you say. Will do asap.